src/components/Book/Book.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable */
```

Comment:
- [ ] Please avoid disabling all linter errors. you should try to avoid any linter errors in general and only disabling specific errors in a situation where it is absolutely necessary.

Reply:
Thank you for your review. I have fixed this issue.